### PR TITLE
fix forward Error message spelling

### PR DESCRIPTION
**Pull request by dcousens**

### CODE REVIEW

**Comment by dcousens** [comment]
@Prestaul not sure if you realised, but this still hasn't been merged :)

File: index.js [hunk 1]
```diff
@@ -40,8 +40,8 @@ function jsend(config, host) {
 		if(json.status === 'success')
 			done(null, json.data);
 		else {
-			var err = new Error(json.message || ('Jsend repsonse status: ' + json.status));
-			if('code' in json) err.code = json.code;
+			var err = new Error(json.message || ('Jsend response status: ' + json.status));
+			if ('code' in json) err.code = json.code;
```
Comment (cdhamm) [inline index.js]: This space doesn't fit with the formatting conventions throughout the rest of the file.  Recommend preserving consistency and reverting this line.

:+1: to the spelling fix

Comment (dcousens) [inline index.js]: Sure

**Comment by dcousens** [comment]
ping @Prestaul

**Comment by Prestaul** [comment]
@dcousens I don't know how I missed this. Thanks for pinging me!

**Comment by Prestaul** [comment]
Published in v1.0.1

**Pull request merged**